Import files 2

sbesson opened the pull request:
These changes

- update the list of import file paths initially introduced in #1 to match #4 i.e. only load the relevant subset of the data as images. All remaining files are labelled masks which will be handled as attachment and converted as ROIs.
- add rendering settings for each dataset activating the LaminB1 and Dapi channels as blue and yellow to match the Figure 3 of the publication

sbesson committed:
Update filePaths.tsv to match annotation.csv

sbesson committed:
Fix filepaths

sbesson committed:
Fix target

sbesson committed:
Add first version of rendering settings configuration

Activate Dapi + LaminB1 using yellow/blue to match the Figure 3 of the publication

sbesson committed:
Fix colors and active channels

sbesson commented:
Images re-imported and rendered on `test64`

<img width="1292" alt="Screen Shot 2019-06-04 at 15 08 33" src="https://user-images.githubusercontent.com/1355463/58885751-a6be6980-86da-11e9-976c-724335084bf8.png">

manics commented:
I think the max for the blue channel should be reduced from the default 4095 (e.g. halve the range to 0-2047?), otherwise you can barely see the blue channel at all in most images, but I'll let @francesw have the final say.

sbesson committed:
Improve contrast in rendering settings

sbesson committed:
Merge remote-tracking branch 'origin/master' into import_files_2

sbesson committed:
Skip all import steps

As we are imposing the rendering settings and one of the image generates a
pyramid, it is easier to skip the thumbnails.

sbesson committed:
Remove original study file
